agent/agent-tooling/spotbugs.exclude.xml

```diff
@@ -57,5 +57,49 @@
         <Class name="com.microsoft.applicationinsights.agent.internal.processors.AttributeProcessor" />
         <Method name="procesHashAction" />
     </Match>
-
+    <Match>
+        <Bug pattern="PATH_TRAVERSAL_IN" />
+        <Or>
+            <And>
+                <!-- False Positive. Path is not from an external source. -->
+                <Class name="com.microsoft.applicationinsights.agent.internal.wasbootstrap.MainEntryPoint" />
+                <Method name="start"/>
+            </And>
+            <And>
+                <!-- False Positive. Path is not from an external source. -->
+                <Class name="com.microsoft.applicationinsights.agent.internal.wasbootstrap.MainEntryPoint" />
+                <Method name="logErrorMessage"/>
+            </And>
+        </Or>
+    </Match>
+    <Match>
+        <Bug pattern="UUF_UNUSED_PUBLIC_OR_PROTECTED_FIELD" />
+        <!-- used from agent-tooling -->
+        <Class name="~com\.microsoft\.applicationinsights\.agent\.internal\.wasbootstrap\.configuration\..*" />
+    </Match>
+    <Match>
+        <Bug pattern="URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD" />
+        <!-- used from agent-tooling -->
+        <Class name="~com\.microsoft\.applicationinsights\.agent\.internal\.wasbootstrap\.configuration\..*" />
+    </Match>
+    <Match>
+        <Bug pattern="UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD" />
+        <!-- used from agent-tooling -->
+        <Class name="~com\.microsoft\.applicationinsights\.agent\.internal\.wasbootstrap\.configuration\..*" />
+    </Match>
+    <Match>
+        <Bug pattern="CRLF_INJECTION_LOGS" />
+        <Class name="com.microsoft.applicationinsights.agent.internal.wasbootstrap.MainEntryPoint" />
```

Comment:
rename `wasbootstrap`

Reply:
I will rename in separate PR, it's an internal package and so many changes in this PR already